graphsync.go

```diff
@@ -126,17 +126,30 @@ func (e RequestNotFoundErr) Error() string {
 	return "request not found"
 }
 
-// RemoteMissingBlockErr indicates that the remote peer was missing a block
-// in the selector requested. It is a non-terminal error in the error stream
+// MissingBlockErr indicates that the remote peer was missing a block
+// in the selector requested, and we also don't have it locally.
+// It is a non-terminal error in the error stream
 // for a request and does NOT cause a request to fail completely
-type RemoteMissingBlockErr struct {
+type MissingBlockErr struct {
 	Link ipld.Link
 }
 
-func (e RemoteMissingBlockErr) Error() string {
+func (e MissingBlockErr) Error() string {
 	return fmt.Sprintf("remote peer is missing block: %s", e.Link.String())
 }
 
+// RemoteIncorrectResponseError indicates that the remote peer sent a response
+// to a traversal that did not correspond with the expected next link
+// in the selector traversal based on verification of data up to this point
+type RemoteIncorrectResponseError struct {
```

Comment:
the path where this link mismatched occured might be useful context as well

Reply:
done